### PR TITLE
feat(combo): add simplified combo sample

### LINKED ISSUE #372: Single Selection Combo Sample

### PR DESCRIPTION
**Pull request by simeonoff**
Closes #372

### CODE REVIEW

File: samples/grids/combo/simplified/index.html [hunk 1]
```diff
@@ -0,0 +1,32 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <title>Combo Features</title>
```
Comment (SisIvanova) [inline samples/grids/combo/simplified/index.html:4]:
```suggestion
    <title>Combo Simplified</title>
```

File: samples/grids/combo/simplified/package.json [hunk 2]
```diff
@@ -0,0 +1,58 @@
+{
+  "name": "wc-combo-features",
```
Comment (SisIvanova) [inline samples/grids/combo/simplified/package.json:2]:
```suggestion
  "name": "wc-combo-simplified",
```

File: samples/grids/combo/simplified/package.json [hunk 3]
```diff
@@ -0,0 +1,58 @@
...
+  "version": "1.0.0",
+  "description": "This project provides example of Combo Features using IgniteUI for Web Components",
```
Comment (SisIvanova) [inline samples/grids/combo/simplified/package.json:4]:
```suggestion
  "description": "This project provides example of Combo Simplified using IgniteUI for Web Components",
```

File: samples/grids/combo/simplified/ReadMe.md [hunk 4]
```diff
@@ -0,0 +1,51 @@
+<!-- NOTE: do not change this file because it's auto re-generated from template: -->
+<!-- https://github.com/IgniteUI/igniteui-wc-examples/tree/vnext/templates/browser/sample/ReadMe.md -->
+
+This folder contains implementation of Web Components application with example of Features feature using [Combo](https://infragistics.com/webcomponentssite/components/general-getting-started.html) component.
```
Comment (SisIvanova) [inline samples/grids/combo/simplified/ReadMe.md:4]:
```suggestion
This folder contains implementation of Web Components application with example of Simplified feature using [Combo](https://infragistics.com/webcomponentssite/components/general-getting-started.html) component.
```

File: samples/grids/combo/simplified/ReadMe.md [hunk 5]
```diff
@@ -0,0 +1,51 @@
...
+</html>
+
+## Instructions
+
+To set up this project locally, execute these commands:
+
+```
+git clone https://github.com/IgniteUI/igniteui-wc-examples.git
+cd ./igniteui-wc-examples
+cd ./samples/grids/combo/features
```
Comment (SisIvanova) [inline samples/grids/combo/simplified/ReadMe.md:31]:
```suggestion
cd ./samples/grids/combo/simplified
```

**Review by SisIvanova — DISMISSED** [review]

**Review by HUSSAR-mtrela — APPROVED** [review]

**Comment by HUSSAR-mtrela** [comment]
@simeonoff please resolve above conflicts

**Review by HUSSAR-mtrela — APPROVED** [review]

**Pull request merged**